src/main/webapp/resources/js/apis/export/ncbi.ts

```diff
@@ -0,0 +1,63 @@
+import axios from "axios";
+import { NcbiSubmission, UserMinimal } from "../../types/irida";
+import { setBaseUrl } from "../../utilities/url-utilities";
+import { ExportUploadState } from "../../types/irida/ExportUpoadState";
+
+const BASE_URL = setBaseUrl(`/ajax/ncbi`);
+
+export interface NcbiExportSubmissionTableModel {
+  exportedSamples: number;
+  state: ExportUploadState;
+  submitter: UserMinimal;
+  bioProjectId: string;
+}
+
+/**
+ * Fetch a list of all NCBI SRA Submission for a given project
+ * @param projectId - Identifier for the current project
+ */
+export async function getProjectNCBIExports(
+  projectId: number
+): Promise<NcbiExportSubmissionTableModel[]> {
+  try {
+    const { data } = await axios.get(`${BASE_URL}/project/${projectId}/list`);
+    return data;
+  } catch (error) {
+    if (axios.isAxiosError(error)) {
+      if (error.response) {
+        return Promise.reject(error.response.data.error);
+      } else {
+        return Promise.reject(error.message);
+      }
+    } else {
+      return Promise.reject("An unexpected error occurred");
```

Comment:
Would this end up being displayed in the UI if it was hit? If so please internationalize it

Reply:
Updated in 98d59fa.  This should never be reached but your right, it should be internationalized.  This is also updated in the next branch where there will only be one place for this.